e2e/container/run_test.go

```diff
@@ -280,3 +282,57 @@ func TestProcessTermination(t *testing.T) {
 		ExitCode: 0,
 	})
 }
+
+// Adapted from https://github.com/docker/for-mac/issues/7632#issue-2932169772
+// Thanks https://github.com/almet!
```

Comment:
Smallest of all nits:
```suggestion
// Thanks https://github.com/almet !
```
or 
```suggestion
// Thanks [@almet](https://github.com/almet)!
```
Otherwise the `!` will read as a part of the link 😅

Reply:
Ah, thanks, fixed!